boot/zephyr/serial_adapter.c

```diff
@@ -53,138 +53,157 @@ static int boot_uart_fifo_init(void);
 int
 console_out(int c)
 {
-    uart_poll_out(uart_dev, c);
+	printk("tx: %d\n", c);
```

Comment:
Is printk()'s ok here and in console_read()?

Reply:
Nope, thanks for catching this. This was a mistake. I have removed this now